halotools/mock_observables/pair_counters/sinha_pairs/sinha_pairs_wrapper.pyx

```diff
@@ -0,0 +1,192 @@
+from countpairs_pbc cimport countpairs_pbc as countpairs_pbc
+from countpairs_nopbc cimport countpairs_nopbc as countpairs_nopbc
```

Comment:
is there a reason for the ``as`` parts?  it looks like this is exactly the same if you left off the `` as ...`` parts.